src/Helpers/validateHelpers.tsx

```diff
@@ -0,0 +1,16 @@
+export const validateEmail = (email?: string) => {
```

Comment:
No usage

Reply:
https://github.com/trucknet-io/react-native-web-uikit/pull/49/files#diff-b04853d66b6be74af361783d5defdb8f

Reply:
Then it is better to colocate it. Read `"Reusable" utility files` section of [this](https://kentcdodds.com/blog/colocation) article.